src/invoices/cc-invoice-list.js

```diff
@@ -140,6 +155,22 @@ export class CcInvoiceList extends LitElement {
         cc-toggle {
           justify-self: start;
         }
+
+        :host([w-lt-520]) cc-toggle {
+          display: none;
+        }
+
+        cc-select {
+          display: flex;
+          gap: 0.5em;
+          margin-left: auto;
+          white-space: nowrap;
+          width: max-content;
+        }
```

Comment:
This relies on the fact that you know the `cc-select` has the label and input as direct children. Because there's also a third child, the gap introduces a space on the right and breaks alignment.

I think this should be handled by `cc-select` component directly.

We could add a `inline` property maybe, this needs to be discussed briefly.